samples/tab-deeplink/csharp/DeepLinkBot/Manifest/manifest.json

```diff
@@ -30,6 +30,14 @@
       "scopes": [
         "groupchat",
         "team"
+      ],
+      "context": [
+        "channelTab",
+        "privateChatTab",
+        "meetingSidePanel",
+        "meetingStage",
+        "meetingChatTab",
```

Comment:
Do we need all of these context ?

Reply:
Yes, only one we can remove but I keep it, so the user doesn't get confused about app installation.